core/src/main/python/update.py

```diff
@@ -184,6 +201,81 @@ def __process_encryption_args(optional_arg_map):
         optional_arg_map[CommandLineArgUtil.PASSPHRASE_SWITCH] = String(passphrase)
     return
 
+# def __process_update_rcu_schema_password_args(model, model_context, aliases):
```

Comment:
Why is it necessary to leave all these commented sections of code? Why not remove them?

Reply:
I will remove the commented out code